source/sass/components/_social-list.scss

```
@@ -2,4 +2,21 @@
   list-style: none;
   display: flex;
   justify-content: center;
+  border-bottom: 1px solid $border_color_light-grey;
+  margin: 0;
+  padding: 0;
+
+  &--intends {
+    padding: 25px 0 14px 0;
+  }
+}
+
+.social-list__item {
+  &:first-child {
```

Как-то очень заумно с детьми, попробуй так: 
```suggestion
  &:not(:last-child) {
    margin-right: 30пк
  }
```
и nth-child удали

---

Но 30пк плохо для резиновой верстки, надо в %